recipes/flang-rt/meta.yaml

```diff
@@ -0,0 +1,91 @@
+{% set version = "21.0.0.dev1" %}
+{% set major = version.split(".")[0] %}
+
+package:
+  name: flang-rt-split
+  version: {{ version }}
+
+source:
+  # url: https://github.com/llvm/llvm-project/archive/refs/tags/llvmorg-{{ version.replace(".rc", "-rc") }}.tar.gz
+  url: https://github.com/llvm/llvm-project/archive/e5129b7e2092c54e503f60cc0ac552c765d5a144.tar.gz
+  sha256: f8506cf5d53a5d555384a8917432db40ef8a2b89a5172d64f8a753b649f0274b
+  patches:
+    - patches/0001-avoid-baking-in-build-time-paths-to-compiler-rt-into.patch
+
+build:
+  number: 0
+  # intentionally only windows (main target) & linux (debuggability)
+  skip: true  # [osx]
+
+requirements:
+  build:
+    - cmake
+    - ninja
+    - {{ stdlib('c') }}
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - flang {{ version }}.*
+  host:
+    - clangdev =={{ version }}
+    - compiler-rt =={{ version }}
+    - llvmdev =={{ version }}
+    - mlir =={{ version }}
+
+outputs:
+  - name: libflang-rt
+    build:
+      # not supported on windows yet, see
+      # https://github.com/llvm/llvm-project/issues/134186
+      skip: true  # [win]
+    files:
+      - lib/libflang_rt.runtime.so  # [linux]
+    requirements:
+      build:
+        # for strong run-exports
+        - {{ stdlib('c') }}
+      run_constrained:
+        # avoid old flang runtime
+        - libflang <0.0a0
+    test:
+      commands:
+        # shared library; copied from target-specific location
+        - test -f $PREFIX/lib/libflang_rt.runtime.so    # [linux]
+
+  - name: flang-rt_{{ target_platform }}
+    build:
+      noarch: generic
+    files:
+      - lib/clang/{{ major }}/lib/x86_64-unknown-linux-gnu/libflang_rt.runtime.a                # [linux]
+      - lib/clang/{{ major }}/lib/x86_64-unknown-linux-gnu/libflang_rt.runtime.so               # [linux]
+      - Library/lib/clang/{{ major }}/lib/x86_64-pc-windows-msvc/flang_rt.runtime.static.lib    # [win]
+    requirements:
+      build:
+        # for strong run-exports
+        - {{ stdlib('c') }}     # [linux]
```

Comment:
Noarch: generic does not add strong run_exports, nor is it necessary.

Reply:
The link check fails for the so if the sysroot isn't found.

Reply:
See [logs](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=1218231&view=results) for https://github.com/conda-forge/staged-recipes/pull/29582/commits/259ddb087cf15d5a8fe6bb6113c73cf9ad712df1

Reply:
I tried again, and it still fails
```
  ERROR (flang-rt_linux-64,lib/clang/21/lib/x86_64-unknown-linux-gnu/libflang_rt.runtime.so): /lib64/libm.so.6 not found in packages, sysroot(s) nor the missing_dso_whitelist.
.. is this binary repackaging?
  ERROR (flang-rt_linux-64,lib/clang/21/lib/x86_64-unknown-linux-gnu/libflang_rt.runtime.so): /lib64/libpthread.so.0 not found in packages, sysroot(s) nor the missing_dso_whitelist.
.. is this binary repackaging?
  ERROR (flang-rt_linux-64,lib/clang/21/lib/x86_64-unknown-linux-gnu/libflang_rt.runtime.so): /lib64/libc.so.6 not found in packages, sysroot(s) nor the missing_dso_whitelist.
.. is this binary repackaging?
```

Reply:
AFAICT, this is inherently a problem of wanting to put an arch-specific SO into a (morally) noarch output. I guess the easiest is to just remove the noarch?

Reply:
OK, I reverted https://github.com/conda-forge/staged-recipes/pull/29582/commits/d2cc694fb52301e19601e74188980d42b89e8d15